packages/ember-data/lib/adapters/rest-adapter.js

```diff
@@ -838,3 +838,13 @@ function endsWith(string, suffix) {
     return string.endsWith(suffix);
   }
 }
+
+Ember.defineProperty(RESTAdapter, 'maxUrlLength', {
```

Comment:
@bmac I saw this too, so that basically mean that people developping in a world without property accessors, they just don't have the deprecation warning, right ?